translations/ssofact.de.po

```diff
@@ -0,0 +1,177 @@
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:165 modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:108 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:108
+msgid "Log in"
+msgstr "Einloggen"
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:123 modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php:101
+msgid "Your email address"
+msgstr "Ihre E-Mailadresse."
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:144
+msgid "I accept terms and conditions"
+msgstr "Ich akzeptiere die GeschÃ¤ftsbedingungen"
+
+#: modules/contrib/ssofact/src/Form/SsofactRegisterForm.php:161 modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php:119
+msgid "Sign up"
+msgstr "Anmelden"
+
+#: modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php:137
+msgid "Unable to register your account. Please contact our support."
+msgstr "Ihr Zugangdaten kÃ¶nnen nicht registriert werden. Bitte kontaktieren Sie unseren Support."
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactLoginBlock.php:146
+msgid "Register here"
+msgstr "Hier anmelden"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactLoginBlock.php:151
+msgid "Create a new user account."
+msgstr "Neuen Nutzeraccount anlegen."
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactLoginBlock.php:14
+msgid "User login (ssoFACT)"
+msgstr "Nutzerlogin (ssoFACT)"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactLoginBlock.php:14 modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:14 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:14
+msgid "Forms"
+msgstr "Formulare"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:111 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:111
+msgid "Log in with your existing account."
+msgstr "Melden Sie sich mit Ihrem existierenden Accoung an."
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:119 modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:119
+msgid "Reset your password"
+msgstr "Setzen Sie Ihr Passwort zurÃ¼ck"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:128
+msgid "Continue reading?"
+msgstr "Weiterlesen?"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:131
+msgid "Give us your e-mail address to unlock this post for free"
+msgstr "Bitte geben Sie Ihre E-Mailadresse an, um diesen Beitrag gratis freizuschalten"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php:14
+msgid "User register (ssoFACT)"
+msgstr "Nutzerregistrierung (ssoFACT)"
+
+#: modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php:14
+msgid "User server register (ssoFACT)"
+msgstr "Nutzerregistrierung Server (ssoFACT)"
+
+#: modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:63
+msgid "RFBE key"
+msgstr "RFBE SchlÃ¼ssel"
```

Comment:
All strings starting from here are internal/administrative and should be removed